lib/wpc_proto/internal_modules/proto_config.c

```diff
@@ -276,7 +276,253 @@ bool Proto_config_init(void)
 app_proto_res_e Proto_config_handle_set_config(wp_SetConfigReq *req,
                                                wp_SetConfigResp *resp)
 {
-    return APP_RES_PROTO_NOT_IMPLEMENTED;
+    app_res_e res;
```

Comment:
To mimic the linux gateway behavior, you should automatically stop the stack and restart it automatically at the end if (and only if) one of the setting to be changed required the stack to be stopped to be changed.

Reply:
How could I know which one requires stop ?

Reply:
Done